src/modules/fw_pos_control_l1/FixedwingPositionControl.hpp

```diff
@@ -267,6 +268,25 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
 		FW_POSCTRL_MODE_OTHER
 	} _control_mode_current{FW_POSCTRL_MODE_OTHER};		///< used to check the mode in the last control loop iteration. Use to check if the last iteration was in the same mode.
 
+	enum FW_SPEED_MODE {
+		FW_SPEED_MODE_NORMAL,
+		FW_SPEED_MODE_ECO,
+		FW_SPEED_MODE_DASH,
+	} _speed_mode_current{FW_SPEED_MODE_NORMAL};
+
+	enum FW_SPEED_MODE_COMMANDED {
+		NORMAL,
+		ECO_CRUISE,
+		ECO_FULL,
+		DASH_CRUISE,
+		DASH_FULL
+	} _speed_mode_setting{NORMAL};
```

Comment:
Not used anywhere?

Reply:
It would be enough to just define the enum because the value is captured as a parameter for now.

Reply:
Yep that's a left over from a previous implementation, good catch!